about.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capston project Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/be8db3d534.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-w76AqPfDkMBDXo30jS1Sgez6pr3x5MlQ1ZAGC+nuZB+EYdgRZgiwxhTBTkF7CXvN"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+      </style> 
+  </head>
+<body>
+<article class="about" id="about">
+  <section class="main-container-about">
+    <nav class="first-header">
+      <div class="logo"><a href="index.html">
+        <i style="font-size: 48px" class="fas logo">&#xf7d9;</i></a>
+      </div>
+      <div class="toggle menu">
+        <a href="index.html" class="toggleNav">
+          <i style="font-size: 24px" class="fas">&#xf0c9;</i></a
+        >
+        <div class="exit">
+          <a href="#" class="exit toggleNav">
+            <i style="font-size: 24px" class="fas">&#xf00d;</i>
+          </a>
+        </div>
+        <ul class="desktopMenu">
+          <li class="menuBtn"><a href="#about">About</a></li>
+          <li class="menuBtn"><a href="#program">Program</a></li>
+          <li class="menuBtn"><a href="#join">Join</a></li>
+          <li class="menuBtn"><a href="#partners">Sponsor</a></li>
+          <li class="menuBtn"><a href="#news">News</a></li>
+        </ul>
```

Comment:
- [ ] I totally understand that to navigate to the `Home` page we click on the logo but I highly recommend always adding the `home` link to navigate to the homepage in the navigation bar instead of the `About` link for a better user experience.